newclient/src/core/components/ui/AppFootnotes.vue

```diff
@@ -43,7 +43,7 @@ export default class AppFootnotes extends Vue {
     }
 
     get isMobile(): boolean {
-        return this.$vuetify.breakpoint.name === 'sm'
+        return this.$vuetify.breakpoint.name === 'sm' || $vuetify.breakpoint.name === 'xs'
```

Comment:
should this be: 
`return this.$vuetify.breakpoint.name === 'sm' || this.$vuetify.breakpoint.name === 'xs'` ?

Reply:
oh yes